libqtile/bar.py

```diff
@@ -309,6 +310,14 @@ def _configure(self, qtile, screen, reconfigure=False):
                 if success:
                     qtile.register_widget(i)
 
+        # Alert the user that we've renamed some widgets
+        if self.qtile.renamed_widgets:
+            logger.info(
+                "The following widgets were renamed in qtile.widgets_map: %s "
+                "To bind commands, rename the widget or use lazy.widget[new_name].",
+                ", ".join(self.qtile.renamed_widgets),
+            )
+
```

Comment:
Could we add a `self.qtile.renamed_widgets.clear()` here?

Reply:
Ha. You beat me to it. I was just adding it!